### PR TITLE
Statistics

### PR DESCRIPTION
**Pull request by TihomirovPavel**
### Checklist
- [x] I've run `mvn test` from the root directory to see all new and existing tests pass
- [x] I've followed code style and run and ensured the code style is valid
- [x] I've created new tests if necessary.

### Motivation and Context
#63 #59 

### Description
just small final fixes, tests

### CODE REVIEW

**Review by kristaps-lipsha-ctco — DISMISSED** [review]

File: src/main/webapp/app/js/commons.js [hunk 1]
```diff
@@ -50,10 +50,35 @@ function switchPersonalData() {
...
+                completed++;
+                break;
+            case "OVERDUE":
+                overdue++;
+                break;
+            default:
+                open++;
+        }
+    }
+    var obj = "<table>" +
```
Comment (olegvm) [inline src/main/webapp/app/js/commons.js]: Should have a clearer name, such as `statisticsTable`.

File: src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java [hunk 2]
```diff
@@ -452,4 +452,41 @@ private GoalFormDto setupGoalFormDto() {
...
+                .thenReturn(user1);
+        when(goalStore.getGoalById(id))
+                .thenReturn(Optional.of(goal));
+        goalApi.setStatusAchieved(id);
+        assertThat(goal.getStatus(), is(GoalStatus.ACHIEVED));
+    }
+
+    @Test
+    @DisplayName("test setStatusAchieved: throws ValidationException wrong user")
+    void testSetStatusAchievedThrowsExceptionIfWrongUser() {
```
[the remark this answers is not in the document]
Comment (TihomirovPavel) [reply src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java]: ok. will remember should i do it now?

[the remark this answers is not in the document]
Comment (olegvm) [reply src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java]: Yes, better do it, should be quick to do.

**Review by kristaps-lipsha-ctco — APPROVED** [review]

File: src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java [hunk 3]
```diff
@@ -452,4 +452,41 @@ private GoalFormDto setupGoalFormDto() {
         result.setDeadline(LocalDate.of(2918, 1, 1));
         return result;
     }
+
+    @Test
+    @DisplayName("test setStatusAchieved: changes status")
+    void test_SetStatusAchieved() {
```
Comment (olegvm) [inline src/test/java/lv/ctco/javaschool/goal/boundary/GoalApiTest.java]: No need to put the underscore right after "test" - `testSetStatusAchieved` is OK.

**Review by olegvm — APPROVED** [review]

**Pull request merged**